workflow/scripts/preprocess/dada2_trim.R

```diff
@@ -26,23 +25,38 @@ packageVersion("dada2")
 print(paste0("Running multithreaded steps with ", ncores, " cores"))
 sprintf("%s - running FilterAndTrim", Sys.time())
 
-out <- data.frame(filterAndTrim(
-  in_R1, out_R1, in_R2, out_R2,
-  truncLen = filter_trunclen,
-  maxN = 0, maxEE = 2, truncQ = 2, rm.phix = TRUE,
-  compress = TRUE, multithread = ncores
-))
+if (length(in_R1) > 1){
```

Comment:
Might be nice if this was explicitly dependent on the config is paired/is not paired so that if we ever change that in the future it is easy to find all the logic we need to change for that.  Ditto with the toplot array below (potentially these things can even be combined?)

Reply:
Could also potentially do this making uuse of your is_paired() func if possible!

Reply:
done!